src/clib/lib/python/enkf_fs_summary_data.cpp

```diff
@@ -1,9 +1,7 @@
 #include <ert/enkf/enkf_plot_tvector.hpp>
+#include <ert/enkf/ensemble_config.hpp>
```

Comment:
Why?

Reply:
This for example: `Cwrap<ensemble_config_type> ensemble_config`